binaries/localserver/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/scootdev/scoot/local/protocol"
+	"github.com/scootdev/scoot/local/server"
+	"log"
+)
+
+// A Local Scoot server.
+func main() {
+	scootdir, err := protocol.Locate()
+	if err != nil {
+		log.Fatal("Error locating Scoot instance: ", err)
```

Comment:
I had the same comment in an earlier review. It's not entirely intuitive that calling log exits the program.